web/packages/teleport/src/WorkloadIdentity/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ComponentType, useEffect, useRef, useState } from 'react';
+import styled, { useTheme } from 'styled-components';
+
+import { Box, ButtonPrimary, H1, Image, Text } from 'design';
+import { Theme } from 'design/theme';
+import {
+  DetailsTab,
+  FeatureContainer,
+  FeatureSlider,
+} from 'shared/components/EmptyState/EmptyState';
+
+import { FeatureBox } from 'teleport/components/Layout';
+
+import feature1dark from './workload_identity_feature_1_dark_mode-min.svg';
+import feature1light from './workload_identity_feature_1_light_mode-min.svg';
+import feature2dark from './workload_identity_feature_2_dark_mode-min.svg';
+import feature2light from './workload_identity_feature_2_light_mode-min.svg';
+import feature3dark from './workload_identity_feature_3_dark_mode-min.svg';
+import feature3light from './workload_identity_feature_3_light_mode-min.svg';
+
+const maxWidth = '1204px';
+
+export function EmptyState() {
+  const [currIndex, setCurrIndex] = useState(0);
+
+  const PreviewPanel = tabs[currIndex].PreviewPanel;
+
+  const intervalId = useRef<number>();
+  function handleOnClick(clickedIndex: number) {
+    if (intervalId.current !== null) {
+      clearInterval(intervalId.current); // Clear the interval if it exists
+    }
+    setCurrIndex(clickedIndex);
+  }
+
+  useEffect(() => {
+    intervalId.current = window.setInterval(() => {
+      setCurrIndex(latestIndex => (latestIndex + 1) % tabs.length);
+    }, 5000);
+
+    return () => {
+      if (intervalId.current !== null) {
+        clearInterval(intervalId.current); // Cleanup on unmount
+      }
+    };
+  }, []);
+
+  return (
+    <FeatureBox>
+      <Box mt={4} data-testid="tag-empty-state">
```

Comment:
or just remove until we get to testing
```suggestion
      <Box mt={4} data-testid="workload-identity-empty-state">
```

Reply:
ill add a story. there really isnt much testing yet because there is no logic to display/not display it. a story makes sense tho so ill add it, and ill update this to make the testing easier when we do get there. thank you!!